src/Templates/src/templates/maui-mobile/Pages/ProjectDetailPage.xaml

```diff
@@ -118,20 +118,19 @@
                             <Border StrokeShape="RoundRectangle 22" 
                                 HeightRequest="44" 
                                 VerticalOptions="Center"
-                                BackgroundColor="Transparent"
                                 Padding="{OnPlatform '18,0,18,8',Android='18,0,18,0'}"
                                 SemanticProperties.Description="{Binding Title}">
                                 <VisualStateManager.VisualStateGroups>
                                     <VisualStateGroup x:Name="CommonStates">
                                         <VisualState x:Name="Normal">
                                             <VisualState.Setters>
-                                                <Setter Property="Background" Value="{AppThemeBinding Light={StaticResource LightSecondaryBackground},Dark={StaticResource DarkSecondaryBackground}}"/>
+                                                <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource LightSecondaryBackground},Dark={StaticResource DarkSecondaryBackground}}"/>
                                                 <Setter TargetName="TagLabel" Property="Label.TextColor" Value="{AppThemeBinding Light={StaticResource DarkOnLightBackground},Dark={StaticResource LightOnDarkBackground}}"/>
                                             </VisualState.Setters>
                                         </VisualState>
                                         <VisualState x:Name="Selected">
                                             <VisualState.Setters>
-                                                <Setter Property="Background" Value="{Binding DisplayColor}"/>
+                                                <Setter Property="BackgroundColor" Value="{Binding DisplayColor}"/>
```

Comment:
@Vignesh-SF3580 is this a bug? 

Usually Background and BackgroundColor should do the same thing

Reply:
@PureWeen Yes, in View, both Background and BackgroundColor are expected to behave similarly. However, in this specific scenario, we should ignore the CollectionView selection color, as it appears visually odd when selecting tags.
In the VSM, setting the Background property for the selected state causes the selection background to appear along with border background during selection. But when using the BackgroundColor property instead of Background, the selection color is ignored, and only the border background is applied as expected.
Im not sure its bug, but its a workaround to avoid showing the selection color in such cases within the CollectionView.

Reply:
Mnnn, could create an issue to review it? Looks like a bug, as @PureWeen said, both must do the same thing.